.nvmrc

```diff
@@ -0,0 +1 @@
+5
```

Comment:
Should be version 8 instead of 5

Reply:
On production server currently is version 5, when it is upgraded, then we will migrate to the new version installed.